recipes/cudnn/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "7.6.5.32" %}
+
+package:
+  name: cudnn
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cuda_compiler_version }}-linux-x64-v{{ version }}.tgz  # [linux64]
+  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cuda_compiler_version }}-linux-ppc64le-v{{ version }}.tgz  # [ppc64le]
+  sha256: a2a2c7a8ba7b16d323b651766ee37dcfdbc2b50d920f73f8fde85005424960e4  # [linux64 and cuda_compiler_version == "9.2"]
+  sha256: 28355e395f0b2b93ac2c83b61360b35ba6cd0377e44e78be197b6b61b4b492ba  # [linux64 and cuda_compiler_version == "10.0"]
+  sha256: 7eaec8039a2c30ab0bc758d303588767693def6bf49b22485a2c00bf2e136cb3  # [linux64 and cuda_compiler_version == "10.1"]
+  sha256: 600267f2caaed2fd58eb214ba669d8ea35f396a7d19b94822e6b36f9f7088c20  # [linux64 and cuda_compiler_version == "10.2"]
+  sha256: a11f44f9a827b7e69f527a9d260f1637694ff7c1674a3e46bd9ec054a08f9a76  # [ppc64le and cuda_compiler_version == "9.2"]
+  sha256: b1717f4570083bbfc6b8b59f280bae4e4197cc1cb50e9d873c05adf670084c5b  # [ppc64le and cuda_compiler_version == "10.0"]
+  sha256: 97b2faf73eedfc128f2f5762784d21467a95b2d5ba719825419c058f427cbf56  # [ppc64le and cuda_compiler_version == "10.1"]
+  sha256: 7dc08b6ab9331bfd12207d4802c61db1ad7cace7395b67a6e7b16efa0335668b  # [ppc64le and cuda_compiler_version == "10.2"]
+
+build:
+  number: 0
+  skip: True   # [cuda_compiler_version == "None"]
```

Comment:
Tricky: This is not even build on `staged-recipes` at all

Reply:
This is probably harmless since this is just a binary repack. Though for more complicated things agree it is nice to see a build

Historically we've just [hacked a CUDA build in on `staged-recipes`]( https://github.com/conda-forge/staged-recipes/pull/9959/commits/d82b76648393d5e40b549ffe26d111381d2e3562 ). Ideally we would address this with something like PR ( https://github.com/conda-forge/staged-recipes/pull/9804 ), but haven't had much time to sort out the issues there 😞 (help would be very welcome if this is of interest! 😄)